src/Config.js

```diff
@@ -98,13 +100,14 @@ export default class Config {
     return await pkgUp(filePath);
   }
 
-  static async init(filePath: string): Promise<Config> {
+  static async init(filePath: string): Promise<?Config> {
     let fileContents;
     try {
       fileContents = await fs.readFile(filePath);
     } catch (e) {
       if (e.code === 'ENOENT') {
-        logger.error(messages.cannotInitConfigMissingPkgJSON(filePath));
+        // swallow error on missing config
+        return;
```

Comment:
I don't think you should return here, I think you should throw an error and catch that error elsewhere

Reply:
oh interesting, that changes the whole apporach since it is no longer going to return a maybe. 🤔

Reply:
Throw bolt error now, then catching in getPackages. Tests all pass, but I am still slightly uneasy about this mainly because I don't understand the whole flow.